src/components/atoms/AuthCheckBox/index.tsx

```diff
@@ -0,0 +1,30 @@
+import Link from "next/link";
+import { AiFillCheckCircle, AiOutlineCheckCircle } from "react-icons/ai";
+
+interface Props {
+  checked: boolean;
+  onChange: (isChecked: boolean) => void;
+}
+
+function AuthCheckbox({ checked, onChange }: Props) {
```

Comment:
Function Component의 리턴 타입을 적어주시는 편이 좋습니다!

```suggestion
function AuthCheckbox({ checked, onChange }: Props): JSX.Element {
```
이렇게 리턴 타입을 JSX.Element로 적어주시면 좋고, 
컴포넌트를 function 키워드로 선언하지 않고 변수로 선언한다면 FC 라는 react에서 제공되는 타입을 이용할 수도 있습니다.

isLoding이 true인 경우, 함수 중간에 리턴문을 써준다고 가정하면
```tsx
function AuthCheckbox(...) {
  if(isLoading) {
    return
  }
  return <>...</>
}
```
위와 같이 코드를 작성할 수 있습니다. return 문에 <></>를 리턴하지 않고 undefined를 리턴하게 되는데요.
이러면 타입 추론이 JSX.Element | undefined로 됩니다.

그런데 리턴타입을 JSX.Element로 명시한다면 return문에 타입 에러가 나게 됩니다.
```tsx
function AuthCheckbox(...): JSX.Element {
  if(isLoading) {
    return // type error (undefined is not JSX.Element type)
  }
  return <>...</>
}
```

그래서 리턴 타입을 명시했을때 이러한 자잘한 실수들을 방지할 수 있습니다